### PR TITLE
Directory Fix

### PR DESCRIPTION
**Pull request by asmith030**
- Fixing a bug when a directory is passed to --files

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -138,11 +138,11 @@ func run(c *cli.Context) error {
 		}
 		switch stat.IsDir() {
 		case true:
-			files, err := listDirectory(fn)
+			filelist, err := listDirectory(fn)
```
Comment (vaijab) [inline main.go]: `filelist` should be called `fileList` - go naming conventions.

**Comment by vaijab** [comment]
@asmith030 would be great to merge this, could you please squash the commits?

**Comment by gambol99** [comment]
@asmith030 ??

**Comment by gambol99** [comment]
LGTM

**Pull request merged**